inst/rmd/umap_qc.rmd

```diff
@@ -0,0 +1,100 @@
+## Dimensionality Reduction 
+
+```{r message=FALSE}
+# create UMAP colored by number of genes detected 
+scater::plotUMAP(processed_sce,
+                 point_size = 0.3,
+                 point_alpha = 0.5,
+                 colour_by = "detected") +
+  scale_color_viridis_c()+
+  theme_bw() + 
+  guides(
+    color = guide_colorbar(title = "Number of \ngenes detected")
+  )
+```
+
+Compromised cells, calculated using `miQC` described above, are filtered and removed from the library prior to downstream analyses. 
```

Comment:
Should this sentence (and the one after it) appear somewhere else in the report? I feel like we want to include these statements whenever the processed data exists, whether or not the UMAP was calculated.